tests/unit/test_detect_target_brightspot.py

```diff
@@ -0,0 +1,201 @@
+"""
+Test DetectTargetBrightspot module.
+"""
+
+import pathlib
+
+import cv2
+import numpy as np
+import pytest
+
+from modules import detections_and_time
+from modules import image_and_time
+from modules.common.modules.logger import logger
+from modules.detect_target import detect_target_brightspot
+
+
+NUMBER_OF_IMAGES_DETECTIONS = 5
+NUMBER_OF_IMAGES_NO_DETECTIONS = 2
+TEST_PATH = pathlib.Path("tests", "brightspot_example")
+IMAGE_DETECTIONS_FILES = [
+    pathlib.Path(f"ir_detections_{i}.png") for i in range(0, NUMBER_OF_IMAGES_DETECTIONS)
+]
+IMAGE_NO_DETECTIONS_FILES = [
+    pathlib.Path(f"ir_no_detections_{i}.png") for i in range(0, NUMBER_OF_IMAGES_NO_DETECTIONS)
+]
+EXPECTED_DETECTIONS_PATHS = [
+    pathlib.Path(TEST_PATH, f"bounding_box_ir_detections_{i}.txt")
+    for i in range(0, NUMBER_OF_IMAGES_DETECTIONS)
+]
+DETECTION_TEST_CASES = list(zip(IMAGE_DETECTIONS_FILES, EXPECTED_DETECTIONS_PATHS))
```

Comment:
Separate expected and not expected:
```py
TEST_PATH = ...

NUMBER_OF_IMAGES_DETECTIONS = ...
IMAGE_DETECTIONS_FILES = ...
EXPECTED_DETECTIONS_PATHS = ...
DETECTION_TEST_CASES = ...

NUMBER_OF_IMAGES_NO_DETECTIONS = ...
IMAGE_NO_DETECTIONS_FILES = ...

Reply:
Also add: `NO_DETECTION_TEST_CASES = IMAGE_NO_DETECTIONS_FILES` for consistency with the detection test cases.